src/content/community/releases/_index.en.md

```diff
@@ -7,8 +7,7 @@ weight = 40
 
 ## v0.16.2
 
-* Fixed known issue in v0.16.1 where Globalnet did not work properly if deployed using the Helm charts.
```

Comment:
Please check why these two entries are removed.

Reply:
These were never merged to the main branch but we need to start a new section anyway so we can leave it.